docs/source/processes.rst

```diff
@@ -709,6 +721,7 @@ combinations.
 .. |http_scheme| replace:: ``http(s)://``
 .. |s3_scheme| replace:: ``s3://``
 .. |file_scheme| replace:: ``file://``
+.. |vault_ref| replace:: ``vault:<UUID>``
```

Comment:
Why not using the standard URL structure with the vault scheme like this `vault://<UUID>`? That would look neat beside `https://`, `s3://` and `file://` and maybe would be easier to parse since any urlparser could extract the scheme?

Reply:
Good catch. I actually changed it afterwards. It is `vault://<UUID>` currently.